docs/griptape-tools/official-tools/src/structure_run_client_1.py

```diff
@@ -2,13 +2,13 @@
 
 from griptape.drivers import GriptapeCloudStructureRunDriver
 from griptape.structures import Agent
-from griptape.tools import StructureRunClient
+from griptape.tools import StructureRunTool
 
 base_url = os.environ["GRIPTAPE_CLOUD_BASE_URL"]
 api_key = os.environ["GRIPTAPE_CLOUD_API_KEY"]
 structure_id = os.environ["GRIPTAPE_CLOUD_STRUCTURE_ID"]
 
-structure_run_tool = StructureRunClient(
+structure_run_tool = StructureRunTool(
```

Comment:
Not for this PR, would `StructureTool` be unambiguous enough?

Reply:
Maybe, I think it depends what other types of functionality might go into a `StructureTool`